config/dm_config/nnf-dm-config.env

```diff
@@ -7,6 +9,18 @@ dmProgressInterval=5s
 # The interval (in seconds) to supply to the dcp --progress command to dump progress output.
 dcpProgressInterval=1s
 
-# If dmCommand is empty or absent, then use this to override the number of processes
-# used for data movement. If empty or absent, then use the default for this value.
-dmNumProcesses=
+# Additional options to include with dcp command.
+dcpOptions=
+
+# Use this value to pass to the mpirun -np option. If empty, -np will not be used.
+# Note: this value is only used for data movement on the NNF Node. For Lustre-to-Lustre Data
+# Movement, the number of hosts will be used and cannot be overridden here.
```

Comment:
Why not allow this for Lustre-to-Lustre data movement?  I'd expect it to benefit the most from having multiple processes per host since its I/O system call latency will almost certainly be the highest.

Reply:
Brian, this was an artifact from before the switch to host files. It wasn't documented anywhere so I added that information here. Regardless, I will remove this exception.